### PR TITLE
Feat/auth

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
add homepage 
add login page with authContext
add signup page with authContext

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import { useAuth } from "../../context";
+import "./navbar.css";
+
+const Navbar = () => {
+  const {
+    authUser: { token },
+    handleLogout,
+  } = useAuth();
+  console.log("token", token);
```
Comment (shivanigangadharan) [inline src/components/Navbar/Navbar.jsx:12]: console log of token here can be removed

File: src/context/AuthProvider.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  const signupWithUser = async (user) => {
+    try {
+      dispatchAuth({
+        type: AUTH_LOADING,
+        payload: true,
+      });
+      const {
+        data: { foundUser, encodedToken },
+      } = await userSignup(user);
+      // console.log("data sign up data is", data);
```
Comment (shivanigangadharan) [inline src/context/AuthProvider.jsx:79]: can remove this as well if not reqd

File: src/page/Home/home.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+.info-text {
+  width: 80%;
+}
+.auth-btbs {
+  margin-top: auto;
+}
+.auth-signup {
+  width: 8rem;
+  height: 2.4rem;
+  background: #5348c7;
```
Comment (shivanigangadharan) [inline src/page/Home/home.css:34]: should use color variables here to maintain consistency

File: src/page/Home/home.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+.home-info-img {
+  width: 50%;
+}
+.info {
+  margin-left: 4rem;
+  display: flex;
+  padding: 1rem;
+  flex-direction: column;
+  height: 350px;
```
Comment (shivanigangadharan) [inline src/page/Home/home.css:23]: using rem here instead of px could be better

File: src/page/Auth/Signup.jsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    name: "",
+    email: "",
+    password: "",
+  });
+
+  const handleInput = (e) => {
+    const { name, value } = e.target;
+    setUsr({
+      ...user,
+      [name]: value,
```
Comment (shivanigangadharan) [inline src/page/Auth/Signup.jsx:19]: I like the use of dynamic attribute here 👍🏼

**Review by shivanigangadharan — APPROVED** [review]
Please also add a deploy preview for better understanding, overall code looks great, can merge this.

File: src/components/Navbar/navbar.css [hunk 6]
```diff
@@ -0,0 +1,34 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  background-color: #5348c7;
```
Comment (prathmeshjagtap) [inline src/components/Navbar/navbar.css:6]: you can use color variable naming

**Review by prathmeshjagtap — APPROVED** [review]
Else everything is nice and well structured

File: src/page/Home/home.css (hunk 3 above)
Comment (ajay-mandaviya) [reply src/page/Home/home.css:34]: have't decide it which colour  will be using , 
will refactor it  once complete it

**Pull request merged**